apps/vaults/hooks/useSolverCowswap.ts

```diff
@@ -264,7 +285,12 @@ export function useSolverCowswap(): TSolverContext {
 		if (!latestQuote?.current?.quote?.buyAmount || isDisabled) {
 			return (toNormalizedBN(0));
 		}
-		return toNormalizedBN(latestQuote?.current?.quote?.buyAmount, request?.current?.outputToken?.decimals || 18);
+		return (
+			toNormalizedBN(
+				toBigInt(latestQuote?.current?.quote?.buyAmount.toString()),
```

Comment:
Should we extend `toBigInt` to take `BigNumber`? Instead of converting the `BigNumber` to `string`?